Resources/doc/index.rst

```diff
@@ -297,6 +297,11 @@ Here is an example on how to inject the service container into your migrations:
     this bundle will automatically inject the default symfony container into your migration class
     (this because the ``MigrationFactoryDecorator`` shown in this example is the default migration factory used by this bundle).
 
+.. caution::
+
+    The interface ``Symfony\Component\DependencyInjection\ContainerAwareInterface`` has been deprecated in Symfony 6.4 and
+    removed in 7.0. If you use this version or newer, there is currently no way to inject dependencies in migrations.
```

Comment:
I might be nit-picking here, but there has never been a way to inject dependencies into migrations. It was the service container that was injected.